AutoComplete: Fixed alignment

SoyDiego opened the pull request:
Adding this line, fix the alignment issue https://github.com/primefaces/primeng/issues/13532.

### CURRENTLY
![autocomplete problem](https://github.com/primefaces/primeng-sass-theme/assets/19764334/e4e43e0d-ac5f-4426-844f-b00a5dadbb9f)

### AFTER SOLUTION
![autocomplete fixed](https://github.com/primefaces/primeng-sass-theme/assets/19764334/5592d5c0-a4dc-47e2-a89c-8fd13819f573)

mertsincan commented:
Thanks a lot for your contribution!

mertsincan commented:
Sorry for the confusion! I reverted this PR. You need to fix the alignment issues in the core. 

Best Regards,

SoyDiego commented:
> Sorry for the confusion! I reverted this PR. You need to fix the alignment issues in the core.
> 
> Best Regards,

Hi Mert, I don't know how I need to continue with this issue.
I wrote in Discord (**ZeR0ByTe**) but maybe you are busy. I will write here if someone knows the reply.

Always when I did PRs, I changed there but now I realized that the styles are not equal for example in the next screenshot:

![image](https://github.com/primefaces/primeng-sass-theme/assets/19764334/03ff91cb-3f66-40ca-a2c3-6e30bd20a904)

1) ON THE LEFT `primeng-sass-theme` project with SASS has the class `margin-left`
2) ON THE RIGHT `primeng` project has not `margin-left` and also has `cursor`.

Now when we have problems with styles only is necessary update `primeng` project?

I asking because I will do this two issues and with your comment I have doubts now:
https://github.com/primefaces/primeng/issues/13617
https://github.com/primefaces/primeng/issues/13532

I'm blocked until know how is the correct way.

Thanks
